bin/node/src/service.rs

```diff
@@ -245,6 +245,7 @@ pub fn new_authority(
     );
 
     let unit_creation_delay = aleph_config.unit_creation_delay();
+    let unit_saving_path = aleph_config.unit_saving_path();
```

Comment:
Nit: I think I would skip creating those two bindings above and just use the methods: it looks a bit weird when they are declared here and used only once, a few hundred lines below.